etc/cvmfs/osgstorage-auth.conf

```diff
@@ -18,6 +18,7 @@ CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://daejeon-kreonet-net.nationalrese
 CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://dtn-pas.jack.nrp.internet2.edu:8443/"
 CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://osdf-cache.sprace.org.br:8443/"
 CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://mghpcc-cache.nationalresearchplatform.org:8443/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://ds-914.cr.cnaf.infn.it:8443/"
```

Comment:
Please move this down to the list not used by OSG

Reply:
done

Reply:
I'm sorry!  It is actually listed in the Europe section of osg.  So you had it in the right section, although not in the right order.  In fact, it should have been in the egi branch already but #185 and #192 have never been completed.  Let's do this right and finish those instead.

Reply:
The first stage of reorganizing the osg branch was done in #197 but the other branches haven't yet been done.